redis/RedisCluster.php

```diff
@@ -209,10 +209,10 @@ public function msetnx(array $array) { }
     /**
      * Remove specified keys.
      *
-     * @param   int | array $key1 An array of keys, or an undefined number of parameters, each a key: key1 key2 key3
-     *                            ... keyN
-     * @param   string      $key2 ...
-     * @param   string      $key3 ...
+     * @param   int|array|string    $key1 An array of keys, or an undefined number of parameters, each a key: key1 key2 key3
```

Comment:
```
/**
 * Remove specified keys.
 * @param int|string...|array[] $keys
 * @return int Number of keys deleted.
.............
 */
```` 
looks like it's a suitable signature for this case

Reply:
I agree with the first part but why the braces after array?

Reply:
@liamsorsby my mistake - without braces

Reply:
Sorry for the long response. There is a problem with `string...` in docs and tests failed because of it. It's my mistake again, sorry. I believe that the whole signature of this method should be like this - we can delete the third parameter and mark the varidic part in the correct way.
```
    /**
     * Remove specified keys.
     *
     * @param int|string|array $key1 An array of keys, or an undefined number of parameters, each a key: key1 key2 key3
     *                            ... keyN
     * @param int|string ...$otherKeys
     *
     * @return int Number of keys deleted.
     * @link    https://redis.io/commands/del
     * @example
     * <pre>
     * $redisCluster->set('key1', 'val1');
     * $redisCluster->set('key2', 'val2');
     * $redisCluster->set('key3', 'val3');
     * $redisCluster->set('key4', 'val4');
     * $redisCluster->del('key1', 'key2');          // return 2
     * $redisCluster->del(array('key3', 'key4'));   // return 2
     * </pre>
     */
    public function del($key1, ...$otherKeys) { }

```

Reply:
Change it to the above.